content/international-candidates.md

```diff
@@ -52,217 +58,144 @@ keywords:
+### Apply for a place
 
-### Teacher training: skilled worker or student visa?
+<p class="call-to-action__action">
+  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a postgraduate teacher training <span>course</span></a>
+</p>
 
-The type of visa you need will depend on whether you are on a salaried or unsalaried teacher training course.
+Or [search for an undergraduate teacher training course](https://digital.ucas.com/search).
 
-A ‘salaried’ teacher training course means you are paid to work in a school while you train.
+### If you cannot come to England
 
-Salaried training types include:
+You do not necessarily have to be in England to study with an English institution. You can do an international Postgraduate Certificate of Education (PGCEi) outside the UK.
 
-* school direct (salaried)
-* Teach First
-* postgraduate teaching apprenticeships
+A PGCEi will help you develop your teaching skills and knowledge.
 
-An ‘unsalaried’ teacher training course means you pay a fee.
+However, it does not lead to 'qualified teacher status'. This means you will not be able to teach in [the types of schools which require qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts#overview).
 
-Unsalaried courses include:
+## Already a teacher? Teach in England
 
-* higher education institution programmes
-* school-centred initial teacher training (SCITT)
-* school direct (fee-funded)
+Teach in England and you’ll benefit from working in a world-class education system.
 
-[Learn more about ways to train in England](/guidance/become-a-teacher-in-england).
+You’ll earn a [competitive salary](https://getintoteaching.education.gov.uk/salaries-and-benefits), get supported in your professional development and have varied job opportunities.
 
-### Applying for a student visa for an unsalaried teacher training course
+Not looking to teach in England? Consider teaching in [Wales](https://www.discoverteaching.wales/), [Scotland](https://teachinscotland.scot/) or [Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
 
-For an unsalaried teacher training course, you will need a [student visa](https://www.gov.uk/student-visa). Student visas must be sponsored by a Home Office licensed student sponsor.
+### What you need to start teaching
 
-First, [choose your teaching training provider](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+To teach in England you need:
 
-Then, contact your chosen teacher training provider to check they are a Home Office licensed student sponsor – sponsors can help you apply for your visa.
+* teaching skills and experience
+* to pass criminal and professional safeguarding checks (these will be organised by your employer)
+* to [get the right visa or status](/international-candidates#get-the-right-visa-or-status)
```

Comment:
I've seen links that just use the # and then the id, whereas this one has the URL as well. Which is best practice/correct @peteryates

Reply:
When on the same page just the `#id`